lib/Payplug/Authentication.php

```diff
@@ -121,4 +121,47 @@ public static function getPublishableKeys(Payplug $payplug = null)
             return false;
         }
     }
+
+    /**
+     * Retrieve client id and client_secret_mask from the user manager resource.
+     *
+     * @param   Payplug $payplug the client configuration
+     *
+     * @return  array the client id and client_secret_mask
+     *
+     * @throws  Exception
+     */
+    public static function getClientIdAndSecretMask(Payplug $payplug = null, $session = null)
+    {
+        if ($payplug === null) {
+            $payplug = Payplug::getDefaultConfiguration();
+        }
+        $kratosSession = self::setKratosSession($session);
+
+        $httpClient = new Core\HttpClient($payplug);
+        $response = $httpClient->get(Core\APIRoutes::$USER_MANAGER_RESOURCE, null, $kratosSession);
+        $result = array();
+
+        foreach ($response['httpResponse'] as $client) {
+            $result[] = array(
+                'client_id' => $client['client_id'],
+                'client_secret_mask' => $client['client_secret_mask']
+            );
+        }
+
+        return $result;
```

Comment:
I think it's best to do this kind of call in a try catch to get, at least an exception if something goes wrong with the request.
Actually, and I don't speak only for this method but for a big part of this lib, we don't have any treatment if something goes wrong with HTTP requests;

Reply:
Here we don't need to put an extra try/catch since it's done inside the request function that get calls